src/main/app/src/app/shared/material-form-builder/form-components/radio/radio.component.html

```diff
@@ -0,0 +1,16 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2022 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
+<mat-label [class.required]="data.required">{{data.label | translate}}</mat-label>
+<mat-radio-group class="radio-group"
+                 [formControl]="data.formControl"
+                 [(ngModel)]="selectedValue"
```

Comment:
Is een ngModel en een formcontrol nodig? In principe houden ze beide de waarde bij

Reply:
zonder formcontrol krijgt je geen foutmelding bij het tabben over het veld. dan moet dit handmatig in de component.
model geeft de daadwerkelijke selectie weer
dus vandaar deze oplossing